mlrun/config.py

```diff
@@ -182,6 +186,11 @@ def read_env(env=None, prefix=env_prefix):
             cfg = cfg.setdefault(name, {})
         cfg[path[0]] = value
 
+    # check for mpijob crd version
+    mpijob_crd_version = env.get('MLRUN_MPIJOB_CRD_VERSION')
+    if mpijob_crd_version and not config.get('mpijob_crd_version'):
+        config['mpijob_crd_version'] = mpijob_crd_version
+
```

Comment:
I don't think it's needed, a mapping between config variables and env var is generated automatically